jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -674,9 +674,8 @@ def mk_burn_script(data):
     if not isinstance(data, bytes):
         raise TypeError("data must be in bytes")
     data = binascii.hexlify(data).decode()
-    return (hex(btc.OP_RETURN)[2:]
-        + serialize_script([data])
-    )
+    scr = [btc.OP_RETURN, data]
+    return serialize_script(scr)
```

Comment:
So just to be 100% clear: here, two items passed in, first is an integer, second is hex. Since `serialize_script` only returns binary if at least one of the entries in the list is binary, this will return hex instead. So all seems correct.